src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -338,6 +496,7 @@ private void initializeIntervals() {
      */
     @Override
     public SAMSequenceDictionary getBestAvailableSequenceDictionary() {
+        VCFHeader mergedVCFHeader = new VCFHeader(mergedHeader);
```

Comment:
I think it makes more sense to store the merged VCF header as a field, instead of re-creating it on every call to this method.